dapr/aio/clients/grpc/client.py

```diff
@@ -124,7 +124,8 @@ def __init__(
         """
         useragent = f'dapr-sdk-python/{__version__}'
         if not address:
-            address = f"{settings.DAPR_RUNTIME_HOST}:{settings.DAPR_GRPC_PORT}"
+            address = f'''{settings.DAPR_GRPC_ENDPOINT if settings.DAPR_GRPC_ENDPOINT
```

Comment:
The environment variable affects port too. See logic done in Java SDK - see how it sets port to 80 or 443 automatically based on the URL schema. It needs URL parsing:

https://github.com/dapr/java-sdk/blob/76aec01e9aa4af7a72b910d77685ddd3f0bf86f3/sdk/src/main/java/io/dapr/client/DaprClientBuilder.java#L177C11-L177C11